g2p/api.py

```diff
@@ -6,9 +6,9 @@
 
 from flask import Blueprint, abort
 from flask_cors import CORS
-from flask_restful import Api, Resource, inputs, reqparse
-from networkx.algorithms.dag import ancestors, descendants
-from networkx.exception import NetworkXError
+from flask_restful import Api, Resource, inputs, reqparse  # type: ignore
```

Comment:
`pre-commit` only runs `mypy` on files that have changed - these files happened to change at some point and so the alerts for missing types got triggered.

what we probably should do actually is provide local type stubs for the specific functions/methods we actualy use, but that is a lot more work (and plus we may not use flask much longer anyway)

Reply:
actually we can provide local type stubs that just say "don't check types" for these classes.  I might not put any time into that since flask might be going away...

Reply:
https://mypy.readthedocs.io/en/stable/stubs.html